src/webgl/p5.Geometry.js

```diff
@@ -74,6 +77,43 @@ p5.Geometry = class Geometry {
     }
     return this; // TODO: is this a constructor?
   }
+  // Custom bounding box calculation based on the object's vertices
+  calculateBoundingBox() {
+    if (this.boundingBoxCache) {
+      return this.boundingBoxCache; // Return cached result if available
+    }
+
+    let minVertex = createVector(
```

Comment:
Ah, right, `createVector` called like this assumes p5 is in global mode, but it needs to also work in instance mode (where you have a `p5` object, and all the previously global functions are referenced via `p5.createVector`.) Rather than trying to detect which mode we're in, maybe instead we can use the constructor syntax, which should work regardless of mode:
```js
let minVertex = new p5.Vector(Number.MAX_VALUE, Number.MAX_VALUE, Number.MAX_VALUE);
```